config/validate.go

```diff
@@ -184,6 +184,17 @@ func validateDatatype(k string, v any, typ string) string {
 		if u.Scheme != "http" && u.Scheme != "https" {
 			return fmt.Sprintf("field %s (%v) must use an http or https scheme", k, v)
 		}
+	case "defaulttrue":
+		switch val := v.(type) {
+		case bool:
+			return ""
+		case string:
+			if val != "t" && val != "f" && val != "true" && val != "false" {
```

Comment:
Please use `strings.ToLower(val)`. Another way to do this if you prefer would be something like:
```
switch strings.ToLower(val) {
case "t", "true", "f", "false":
default:
    // error
```

Reply:
Sure, the second switch statement looks better to me.